hackIT/test_challenges/boxes/2.a

```diff
@@ -0,0 +1 @@
+9417, 1324220799951242036592010149888000, -8819
```

Comment:
The second number is larger than the max interger size (2^64 - 1) in java. So a) change to a number under 2^64 -1 or b) give resource to how one can use larger numbers in java